### ISSUE TITLE
Hi, thanks for your open codes, I want to know how to get the "topo_label" in this loss forward?

### ISSUE THREAD

**Issue by gentlezr** [mmmmimic/Topo-Loss#2]

**Comment by mmmmimic** [comment]
Hi, to utilize the topoloss, you may need to have "topo_label" (manually) annotated in your dataset. In the [https://github.com/JamesClough/topograd/blob/master/MNIST_example.ipynb](official implementation), MINIST was used as an example. For different numbers, the label was constructed as:
`
H_1 = {0:1, 1:0} # 1, 2, 3, 4, 5, 7;

H_0 = {0:1, 1:1} # 0, 6, 9;

H_8 = {0:1, 1:2} # 8;

H_dict = {0:H_0,
          1:H_1,
          2:H_1,
          3:H_1,
          4:H_1,
          5:H_1,
          6:H_0,
          7:H_1,
          8:H_8,
          9:H_0}
`

For each number, two values were used as labels, one of which means the number of connected components and the other means the number of holes. Take 8 for example, there are 1 connected component, and 2 holes, thus its label is (1,2). 

In this implementation, we consider multi-class segmentation. The topo_label should be of shape (B, C, 2), where B is the batch size, C is the number of categories (excluding the background category), and 2 is the label value mentioned before. Say we have an image presenting '8', which belongs to category 2. The image is the 5th image in the batch of size 16, and there are 8 foreground categories (9 classes in total including background) in the task. The label can be constructed by:
`
label = torch.zeros((16, 8, 2), dtype=torch.long)

label[5, 2, 0] = 1

label[5, 2, 1] = 2
`

**Comment by gentlezr** [comment]
Oh， thanks for your reply and it's very clear.
